quipucords/scanner/openshift/entities.py

```diff
@@ -36,6 +36,7 @@ def _update_model_refs():
         model.update_forward_refs()
 
 
+# pylint: disable=too-few-public-methods
```

Comment:
this linting check is already disabled: https://github.com/quipucords/linting/blob/main/base-flakeheaven.toml#L33